Assets/Shopify/UIToolkit/Components/ProductImageHolder.cs

```diff
@@ -0,0 +1,15 @@
+namespace Shopify.UIToolkit {
+    using UnityEngine;
+    using UnityEngine.UI;
+    using Image = UnityEngine.UI.Image;
+
+    public class ProductImageHolder : MonoBehaviour {
```

Comment:
This is pretty much copied from the Example's `ImageHolder` class. We should probably consolidate in the future?